src/code-of-conduct

```diff
@@ -0,0 +1,93 @@
+<!doctype html>
+<html lang="en">
+ <head>
+  <title>Web Hypertext Application Technology Working Group Code of Conduct</title>
+  <link rel="stylesheet" href="/style/tabbed-pages">
+  <link rel="icon" href="https://resources.whatwg.org/logo.svg">
+ </head>
+ <body>
+  <h1>
+   <span class="the">The</span>
+   <strong class="what">Web Hypertext Application Technology</strong>
+   <span class="wg">Working Group</span>
+  </h1>
+  <ul class="navigation">
+   <li><a href="/" rel="home">Home</a></li>
+   <li><a href="https://spec.whatwg.org/">Standards</a></li>
+   <li><a href="/working-mode">Working Mode</a></li>
+   <li class="this"><strong>Code of Conduct</strong></li>
+  </ul>
+
+  <h2>Code of Conduct</h2>
+
+  <h3 id="conduct">Conduct<a class="self-link" href="#conduct"></a></h3>
+
+  <p>Contact: Ian Hickson (Hixie on <a href="https://wiki.whatwg.org/wiki/IRC">IRC</a>,
+  <a href="mailto:ian@hixie.ch">ian@hixie.ch</a> via email) or a member of the community you feel
+  you can trust.
+
+  <ul>
+   <li>We are committed to providing a friendly, safe and welcoming environment for all, regardless
```

Comment:
I think this will look better if each `<li>` also has a `<p>`. I am guessing, based on the spacing at https://whatwg.org/working-mode#removals and in "New Proposals" versus the rest of the document (although probably those sections should not be different from the others in the working mode)